frame/header-mmr/Cargo.toml

```diff
@@ -25,6 +23,8 @@ sp-std = { default-features = false, git = "https://github.com/darwinia-network/
 [dev-dependencies]
 # crates
 serde_json = { version = "1.0.51" }
+# darwinia
+array-bytes = { default-features = true, path = "../../primitives/array-bytes" }
```

Comment:
default-features can be ignore in dev-dependencies.

Reply:
...

Reply:
Because default just mean default... Haha.